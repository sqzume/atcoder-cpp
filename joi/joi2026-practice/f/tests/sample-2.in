6
916
1015
1118
1210
204
324
