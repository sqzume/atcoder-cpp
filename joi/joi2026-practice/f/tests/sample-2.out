324
204
1210
1118
1015
916
